lmdeploy/serve/openai/api_server.py

```diff
@@ -45,7 +45,7 @@ class VariableInterface:
     api_server_url: Optional[str] = None
 
 
-app = FastAPI(docs_url='/')
+router = APIRouter()
```

Comment:
What is the purpose of using `router`?

Reply:
Because the original `app = FastAPI(docs_url='/')` was defined globally, I needed to control the availability of the documentation using the `disable_fastapi_docs` parameter passed into the serve function.

I moved the initialization of `app = FastAPI(...)` into the main function. The endpoints, such as `'/v1/chat/completions'`, are registered through `router`, and finally, I use `app.include_router(router)` to included the router.